src/pages/landing/projects/components/Noteslist.tsx

```diff
@@ -0,0 +1,49 @@
+import { useState } from 'react';
+import { Event } from 'nostr-tools';
+import { Button, Flex } from '@chakra-ui/react';
+import NoteCard from './NoteCard.tsx';
+
+interface Props {
+  projects: Event[];
+}
+
+export default function NotesList({ projects }: Props) {

```

Comment:
Change name to NostrProjectList, for readability and understandability of what is going on here.

Reply:
@sajald77 done the renaming, and other standardization changes requested